frontend/src/components/messstelle/charts/MessstelleDiagramme.vue

```diff
@@ -199,11 +202,17 @@ const zaehldatenStepline: ComputedRef<LadeZaehldatenSteplineDTO> = computed(
     }
 );
 
+const zaehldatenHeatmap: ComputedRef<LadeZaehldatenHeatmapDTO> = computed(
+    () => {
+        return zaehldatenHeatmapDTO.value;
+    }
+);
+
```

Comment:
kannst du hier kurz erklären, warum du das vallue einer ref in eine andere ref speicherst und nicht direkt auf zaehldatenHeatmapDTO zugreifst?